src/app/components/pages/admin-cell/admin-cell.component.html

```diff
@@ -1,5 +1,96 @@
 <div class="admin-cell" *ngIf="cell">
-  <h2>Cellule de {{ cell.name }}</h2>
+  <div class="admin-cell__title">
+    <h2>Cellule de {{ cell.name }}</h2>
+
+  </div>
+
+  <app-my-modal name="create event"
+                title="Création d'un event"
+                button2Label="Créer"
+                (button1)="eventForm.reset()"
+                (button2)="createEvent()"
+                typeModal="form"
+                button2Style="button--success">
+
+    <form [formGroup]="eventForm">
+
+      <input type="hidden" formControlName="cell_id">
+
+      <div class="form-group">
+        <label for="nb_volunteers_needed" class="form-label">Nombre de partcipants requis:</label>
+        <input class="input" formControlName="nb_volunteers_needed" id="nb_volunteers_needed" type="number" autofocus>
+        <div *ngIf="eventForm.controls['nb_volunteers_needed'].hasError('required')"
+             class="form-text form-text--right form-text--danger">Le nombre de volontaires est obligatoire.
```

Comment:
Need to be consistent between `participants` and `volontaires` to be more easy to understand